include/swift/Driver/Job.h

```diff
@@ -34,48 +36,140 @@ namespace driver {
 class Job;
 class JobAction;
 
+/// Some terminology for the following sections (and especially Driver.cpp):
```

Comment:
Nitpick: as a doc comment, this'll get attached to the following declaration. Maybe just make it a normal comment, or maybe it should be a [file-level comment](http://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile)?

Reply:
Done.